android/src/main/java/com/auth0/react/A0Auth0Module.java

```diff
@@ -122,29 +123,35 @@ String generateCodeChallenge(@NonNull String codeVerifier) {
     }
 
     @Override
-    public void onHostResume() {
-        new Handler().postDelayed(new Runnable() {
+    public void onActivityResult(Activity activity, int requestCode, int resultCode, Intent data) {
+        new Handler().post(new Runnable() {
```

Comment:
we probably can get rid of this line. There shouldn't be a need to wrap this on a runnable now that we don't need to post-delayed something.

Reply:
Done in https://github.com/auth0/react-native-auth0/pull/350/commits/4b7998584f638b1b0909a90a37f7a57967121b53.